db/schema.rb

```diff
@@ -264,6 +271,19 @@
     t.index ["service_id"], name: "index_notes_on_service_id"
   end
 
+  create_table "permissions", force: :cascade do |t|
+    t.string "description"
+    t.string "permission", null: false
+    t.string "object_type", null: false
+    t.integer "object_pk", null: false
```

Comment:
It looks like the default column type for primary keys in Rails (with a Postgres backend) is bigint, not, integer, so maybe this should be a bigint to match?

Reply:
Good catch!